pyproject.toml

```diff
@@ -280,6 +280,8 @@ exclude = [
     'dist',
     'doc/examples',
     'doc/_build',
+    'examples',
+    'tests',
```

Comment:
Added to exclude flake8-unused-arguments check, but this setting also excludes other ruff checks which are not good. But there is no other way.

Reply:
```suggestion
    # https://github.com/pyvista/pyvista/pull/5374/files#r1433366947
    'examples',
    'tests',
```